config/default.yml

```diff
@@ -221,6 +221,28 @@ Style/CommentAnnotation:
     - HACK
     - REVIEW
 
+# Checks that you have put a copyright in a comment before any code.
+#
+# You can override the default Notice in your rubocop.yml file.
+#
+# In order to use autocorrect, you must supply a value for the
+# AutocorrectNotice key that matches the regexp Notice.  A blank
+# AutocorrectNotice will cause
+#
+# Autocorrect will add a copyright notice in a comment at the top
+# of the file immediately after any shebang or encoding comments.
+#
+# Example rubocop.yml:
+#
+# Style/Copyright:
+#   Enabled: true
+#   Notice: 'Copyright (\(c\) )?2015 Yahoo! Inc'
+#   AutocorrectNotice: '# Copyright (c) 2015 Yahoo! Inc.'
```

Comment:
`Notice` is a regexp.  `AutocorrectNotice` is a concrete string.

The primary reason notice has to be a regexp is that you might be working on code that has different copyright attribution for different files in the same project.  In these cases, the author generally creating code under one copyright or the other.

It can also be the case that you do not want to trigger an offense just because the year is 2015 but the notice says some previous year.

BTW, this works very similar to the encoding cop, except the encoding cop's notion of 'Notice' is not configurable by the user.

I hope that you will accept this pull request with both these configuration keys.  I am open to other suggestions.